<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataAugmentation</class>
 <widget class="qMRMLWidget" name="DataAugmentation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>855</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>运行时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="time_cost_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Inputs</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <property name="collapsedHeight">
      <number>14</number>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image volume:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSelector">
        <property name="toolTip">
         <string>Pick the input to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>label volume:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="labelSelector">
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>平移旋转</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="title">
         <string>平移变换</string>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>x轴平移</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ctkSliderWidget" name="xTranslationSliderWidget">
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="minimum">
            <double>-100.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
           <property name="suffix">
            <string>mm</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>y轴平移</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>z轴平移</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ctkSliderWidget" name="yTranslationSliderWidget">
           <property name="singleStep">
            <double>0.400000000000000</double>
           </property>
           <property name="minimum">
            <double>-100.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
           <property name="suffix">
            <string>mm</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="ctkSliderWidget" name="zTranslationSliderWidget">
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="minimum">
            <double>-100.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="suffix">
            <string>mm</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="title">
         <string>旋转变换</string>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>x轴旋转</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ctkSliderWidget" name="xRotationSliderWidget">
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
           <property name="suffix">
            <string>°</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>y轴旋转</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ctkSliderWidget" name="yRotationSliderWidget">
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
           <property name="suffix">
            <string>°</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>z轴旋转</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="ctkSliderWidget" name="zRotationSliderWidget">
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
           <property name="suffix">
            <string>°</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>执行操作</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="transformPushButton">
          <property name="text">
           <string>平移旋转变换</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>高斯滤波</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>sigma：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sigma_lineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="gaussBlurPushButton">
        <property name="text">
         <string>GaussBlur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gaussNoisePushButton">
        <property name="text">
         <string>GaussNoise</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>DataAugmentation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DataAugmentation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>labelSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
